core/domain/value_generators_domain.py

```diff
@@ -98,15 +104,15 @@ def _refresh_registry(cls):
                 cls.value_generators_dict[clazz.__name__] = clazz
 
     @classmethod
-    def get_all_generator_classes(cls):
+    def get_all_generator_classes(cls) -> Dict[str, BaseValueGenerator]:
```

Comment:
This does not return a list of generator instances. This returns a list of generator classes. You will need to use `Type[]` here, see https://mypy.readthedocs.io/en/stable/kinds_of_types.html#the-type-of-class-objects

Reply:
Done.